hack/lib/util.sh

```diff
@@ -14,6 +14,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+KUBE_ROOT=$(dirname "${BASH_SOURCE}")/../..
+source ${KUBE_ROOT}/hack/lib/logging.sh
```

Comment:
This shouldn't be needed anymore now that you're sourcing init.sh instead of util.sh directly.